chall01/vkuikka.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python3
+import sys
+
+def my_errorMessage():
+	print ("usage:", sys.argv[0], "<a-zA-Z string>")
+	quit()
```

Comment:
I would suggest changing your print by :

```python
print(f"usage: {sys.argv[0]} <a-zA-Z string>")
```

Plus the good practice about printing usage is : 
- if it's due to an input/user error --> stderr + return something else than 0
- if it's requested by the user (e.g `./app.py --help`) --> stdout + return 0